distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestFederation.java

```diff
@@ -215,7 +215,7 @@ public void testFederatedRetrieveNoProduct() throws Exception {
         String restUrl = REST_PATH + "sources/" + OPENSEARCH_SOURCE_ID + "/"
                 + metacardIds[GEOJSON_RECORD_INDEX] + "?transform=resource";
 
-        expect().log().all().body(containsString("Unknown resource request")).when().get(restUrl);
+        expect().log().all().body(containsString("Try different transformer")).when().get(restUrl);
```

Comment:
Yeah, this should be fixed. No tests should be looking for a particular exception message.